sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Operations.cs

```diff
@@ -158,14 +158,13 @@ public async Task<AzureOperationResponse<IPage<OperationEntity>>> ListWithHttpMe
             string _responseContent = null;
             if ((int)_statusCode != 200)
             {
-                var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
+                var ex = new ErrorException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?